static/app/routes.tsx

```diff
@@ -1505,6 +1508,22 @@ function buildRoutes() {
           )}
         />
       </Route>
+      <Route path={`${PERFORMANCE_MODULE_BASE_URLS[ModuleName.RESOURCE]}/`}>
+        <IndexRoute
+          component={make(
+            () => import('sentry/views/performance/browser/resources/index')
+          )}
+        />
+        <Route
+          path="spans/span/:groupId/"
+          component={make(
+            () =>
+              import(
+                'sentry/views/performance/browser/resources/resourceSummaryPage/index'
+              )
+          )}
+        />
+      </Route>
```

Comment:
`/browser/assets` and `/browser/resources` will just map to the same component so we don't have to rely on the organization to setup routes